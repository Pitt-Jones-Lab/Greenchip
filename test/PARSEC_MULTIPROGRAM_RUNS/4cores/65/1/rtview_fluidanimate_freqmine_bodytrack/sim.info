{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/1/rtview_fluidanimate_freqmine_bodytrack',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_1_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/1/rtview_fluidanimate_freqmine_bodytrack',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpeFOpMY/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb8',
 'pin_version': 'unknown',
 'rusage': (22248.571702000001,
            852.43241,
            201372,
            0,
            0,
            0,
            47940,
            0,
            0,
            16,
            1320,
            0,
            0,
            0,
            70298239,
            6379047),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/1/rtview_fluidanimate_freqmine_bodytrack --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem65.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_1_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpeFOpMY/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/1/rtview_fluidanimate_freqmine_bodytrack/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 11043.925858020782,
 't_start': 1463344392.7117541,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1552580608L}
