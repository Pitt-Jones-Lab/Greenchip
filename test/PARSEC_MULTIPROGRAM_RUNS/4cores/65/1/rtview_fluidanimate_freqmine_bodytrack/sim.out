                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2931614380 |  2701873348 |  2675057629 |  1690363669
  Cycles                           |  2454736578 |  2454736578 |  2454736578 |  2454736578
  IPC                              |        1.19 |        1.10 |        1.09 |        0.69
  Time (ns)                        |   922833300 |   922833300 |   922833300 |   922833300
  Idle time (ns)                   |       97504 |   183728008 |      327538 |   301820099
  Idle time (%)                    |        0.0% |       19.9% |        0.0% |       32.7%
Branch predictor stats             |             |             |             |            
  num correct                      |   353958969 |   317530197 |   374528259 |   168789062
  num incorrect                    |    28761172 |    20844194 |    27071802 |    20321166
  misprediction rate               |       7.51% |       6.16% |       6.74% |      10.75%
  mpki                             |        9.81 |        7.71 |       10.12 |       12.02
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   405303985 |   350183392 |   341253664 |   193264582
    num misses                     |        7947 |         204 |         181 |         411
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |  1041375604 |   944395564 |  1123989865 |   513592923
    num misses                     |      534738 |     1159742 |      652127 |      518755
    miss rate                      |       0.05% |       0.12% |       0.06% |       0.10%
    mpki                           |        0.18 |        0.43 |        0.24 |        0.31
  L2 TLB                           |             |             |             |            
    num accesses                   |      542685 |     1159946 |      652308 |      519166
    num misses                     |      121395 |      328144 |      140791 |      116445
    miss rate                      |      22.37% |      28.29% |      21.58% |      22.43%
    mpki                           |        0.04 |        0.12 |        0.05 |        0.07
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   405303985 |   350183392 |   341253664 |   193264582
    num cache misses               |     6377381 |      976958 |      952591 |      990695
    miss rate                      |       1.57% |       0.28% |       0.28% |       0.51%
    mpki                           |        2.18 |        0.36 |        0.36 |        0.59
  Cache L1-D                       |             |             |             |            
    num cache accesses             |  1041375604 |   944395564 |  1123989865 |   513592923
    num cache misses               |     5099101 |     6426198 |     6635207 |     5294264
    miss rate                      |       0.49% |       0.68% |       0.59% |       1.03%
    mpki                           |        1.74 |        2.38 |        2.48 |        3.13
  Cache L2                         |             |             |             |            
    num cache accesses             |    11795348 |     7723381 |     8111287 |     6596136
    num cache misses               |     1870205 |     2301097 |     2038690 |     1491271
    miss rate                      |      15.86% |      29.79% |      25.13% |      22.61%
    mpki                           |        0.64 |        0.85 |        0.76 |        0.88
DRAM summary                       |             |             |             |            
  num dram accesses                |    11970106 |           0 |           0 |           0
  average dram access latency (ns) |      114.90 |         inf |         inf |         inf
  average dram queueing delay      |      114.90 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     1253670 |     1213424 |     1469986 |     1044056
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
